examples/lang/range.mcl

```diff
@@ -0,0 +1,23 @@
+import "iter"
+import "fmt"
+
+print "range" {
+	msg => fmt.printf("%v", iter.range(1, 200, 3)),
+}
+
+#range (1 2 3) 
```

Comment:
for $i, $x in iter.range(3) {
        print "..." {
                msg => "...",
        }
}